src/theme/DocPage/styles.module.css

```diff
@@ -0,0 +1,104 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+:root {
+  --doc-sidebar-width: 300px;
+  --doc-sidebar-hidden-width: 30px;
+}
+
+:global(.docs-wrapper) {
+  display: flex;
+  margin: 1rem auto;
+  max-width: var(--ifm-container-width);
+  width: 100%;
+}
+
+@media (min-width: 1440px) {
+  :global(.docs-wrapper) {
+    max-width: var(--ifm-container-width-xl);
+  }
+}
+
+.docPage,
+.docMainContainer {
```

Comment:
just would like to know why it is camel case on the class name? or it is from docusaurus itself?

Reply:
These are css module style, used in in the component js file like an object, so better to use camel case here.

Reply:
this is how you can name classes in module css
when page is rendered in browser you can see sth like this
![image](https://user-images.githubusercontent.com/39953699/134657455-27b0f02c-1cdc-42b3-9fad-6a1698f354dd.png)

Reply:
I'm speaking of this: https://github.com/openware/yellow-org/blob/feature/unify-blog-docs/src/theme/DocPage/index.js#L22
And how it is used across the component, example:
https://github.com/openware/yellow-org/blob/feature/unify-blog-docs/src/theme/DocPage/index.js#L22

Reply:
thanks. since we always use BEM in CSS files before. then it's clear